ivy/functional/ivy/experimental/manipulation.py

```diff
@@ -2177,14 +2177,14 @@ def unique_consecutive(
 
 
 @handle_exceptions
-@handle_backend_invalid
 @handle_nestable
+@handle_partial_mixed_function
```

Comment:
No need for this since this is just a mixed function and not a partial mixed function.